doc: clarify about the Node.js-only extensions in perf_hooks (#33199)
nodejs/node: doc/api/perf_hooks.md

# Performance Measurement APIs

<!--introduced_in=v8.5.0-->

> Stability: 2 - Stable

This module provides an implementation of a subset of the W3C
[Web Performance APIs][] as well as additional APIs for
Node.js-specific performance measurements.

> Currently, Node.js supports the following [Web Performance APIs][]:

Reviewer: ```suggestion
Node.js supports the following [Web Performance APIs][]:
```

});
```

## `perf_hooks.performance`
<!-- YAML
added: v8.5.0
-->

An object that can be used to collect performance metrics from the current

> Node.js instance. It's similar to [`window.performance`][] in browsers.

Reviewer: ```suggestion
Node.js instance. It is similar to [`window.performance`][] in browsers.
```

* {PerformanceNodeTiming}

> > Note: This property is an extension by Node.js. It's not available in Web
> > browsers.

Reviewer: ```suggestion
_This property is an extension by Node.js. It is not available in Web browsers._
```

Explanation: Semantically, `>` is for a block quote. We're not quoting anything here. It's being used to highlight/emphasize. I wouldn't be surprised to find that we do this elsewhere, but I'd prefer we use italics for emphasis, or even nothing at all.

* `fn` {Function}

> > Note: This property is an extension by Node.js. It's not available in Web
> > browsers.

Reviewer: ```suggestion
_This property is an extension by Node.js. It is not available in Web browsers._
```

* {string}

> The type of the performance entry. Currently it may be one of:

Reviewer: ```suggestion
The type of the performance entry. It may be one of:
```

* {number}

> > Note: This property is an extension by Node.js. It is not available in Web
> > browsers.

Reviewer: ```suggestion
_This property is an extension by Node.js. It is not available in Web browsers._
```

* {number}

> > Note: This property is an extension by Node.js. It is not available in Web
> > browsers.

Reviewer: ```suggestion
_This property is an extension by Node.js. It is not available in Web browsers._
```

added: v8.5.0
-->

> > Note: This class is an extension by Node.js. It's not available in Web
> > browsers.

Reviewer: ```suggestion
_This property is an extension by Node.js. It is not available in Web browsers._
```

Provides timing details for Node.js itself. The constructor of this class

> is not directly exposed to users.

Reviewer: Is it exposed indirectly? If not:

```suggestion
is not exposed to users.
```

The `PerformanceObserverEntryList` class is used to provide access to the
`PerformanceEntry` instances passed to a `PerformanceObserver`.

> The constructor of this class is not directly exposed to users.

Reviewer: Is it indirectly exposed? If not:

```suggestion
The constructor of this class is not exposed to users.
```

    than zero. **Default:** `10`.
* Returns: {Histogram}

> > Note: This property is an extension by Node.js. It's not available in Web
> > browsers.

Reviewer: ```suggestion
_This property is an extension by Node.js. It is not available in Web browsers._
```

<!-- YAML
added: v11.10.0
-->
Tracks the event loop delay at a given sampling rate. The constructor of

> this class not directly exposed to users.

Reviewer: Is it indirectly exposed? If not:

```suggestion
this class not exposed to users.
```

> > Note: This class is an extension by Node.js. It's not available in Web
> > browsers.

Reviewer: ```suggestion
_This property is an extension by Node.js. It is not available in Web browsers._
```
